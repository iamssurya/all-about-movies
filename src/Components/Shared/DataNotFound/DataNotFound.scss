@import "styles";

.dnf-block {
  margin-top: 40%;
}
