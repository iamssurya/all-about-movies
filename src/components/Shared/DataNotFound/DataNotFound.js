import React from "react";
import { Jumbotron } from "react-bootstrap";

import "./DataNotFound.scss";

const DataNotFound = () => {
  return (
    <Jumbotron className="mt-3 bg-dark text-white">
      <h1>Oops!</h1>
      <p>The information you have requested is not found!</p>
    </Jumbotron>
  );
};

export default React.memo(DataNotFound);
