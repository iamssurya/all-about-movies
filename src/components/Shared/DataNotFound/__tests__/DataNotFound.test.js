import React from "react";
import Enzyme, { mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import DataNotFound from "../DataNotFound";

Enzyme.configure({ adapter: new Adapter() });

describe("<DataNotFound /> component tests", () => {
  let Component;
  let Wrapper;
  beforeEach(() => {
    Component = mount(<DataNotFound />);
    Wrapper = toJson(Component);
  });

  it("Should render component correctly", () => {
    expect(Wrapper).toMatchSnapshot();
  });
});
