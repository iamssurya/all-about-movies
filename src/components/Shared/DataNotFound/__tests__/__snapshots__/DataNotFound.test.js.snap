// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DataNotFound /> component tests Should render component correctly 1`] = `
<Memo(DataNotFound)>
  <Jumbotron
    className="mt-3 bg-dark text-white"
    fluid={false}
  >
    <div
      className="mt-3 bg-dark text-white jumbotron"
    >
      <h1>
        Oops!
      </h1>
      <p>
        The information you have requested is not found!
      </p>
    </div>
  </Jumbotron>
</Memo(DataNotFound)>
`;
